test/packages/good_v2/manifest.yml

```diff
@@ -1,4 +1,4 @@
-format_version: 2.10.0
+format_version: 2.12.0
```

Comment:
Should it be added here in `good_v2` also the new `agent` block ?

```yaml
agent:
  privileges:
    root: true
```

Reply:
Added :+1: